pom.xml

```diff
@@ -139,7 +139,7 @@
         <dependency>
             <groupId>com.oracle.database.jdbc</groupId>
             <artifactId>ojdbc8</artifactId>
-            <version>23.2.0.0</version>
+            <version>19.20.0.0</version>
```

Comment:
Hi @rberezen 
Could you please add a comment that the 19.x version is LTS right now and it shouldn't be updated to not LTS?
https://oracledbwr.com/oracle-release-schedule-of-current-database-releases/
I can't find the date of the end of this LTS version support, unfortunately.

Reply:
sorry @vitaliimak 
I am late, the PR has been merged
do you want me to create another PR?

Reply:
@vitaliimak @rberezen as a suggestion: dependabot will open a new PR today/tomorrow to upgrade again to 23.2 . You can close the PR asking dependabot to ignore major version 23 going forward, then we don't need to worry about it and we still get updates for 19 series.

Reply:
@rberezen No, another PR isn't needed. Thank you.